libnavigation-base/src/main/java/com/mapbox/navigation/base/options/HistoryRecorderOptions.kt

```diff
@@ -3,7 +3,8 @@ package com.mapbox.navigation.base.options
 /**
  * Defines options for recording history files.
  *
- * @param fileDirectory used for saving history files. Null use a default directory.
+ * @param fileDirectory used for saving history files. If null is provided a default directory
+ * will be used ("mbx_nav/history").
```

Comment:
`("mbx_nav/history")` doesn't provide much information. You still have to dig into the implementation in order to figure out if it is in `cacheDir` or `filesDir`

Also it creates confusion for this property, `fileDirectory` is an absolute path not a relative path.

Reply:
> You still have to dig into the implementation in order to figure out if it is in cacheDir or filesDir

what for do you need to figure it out? customer doesn't care if it's CACHE or HISTORY in SDK terminology, but it's a good idea to say what dir will be used if no path is provided. And ("mbx_nav/history") will be used

Reply:
returned old description